mwcites/extractors/tests/test_doi.py

```diff
@@ -2,8 +2,8 @@
 
 from nose.tools import eq_
 
-from .. import doi
-from ...identifier import Identifier
+from mwcites.extractors import doi
+from mwcites.identifier import Identifier
```

Comment:
It's important to do relative imports explicitly.   I'll clean this up and get it merged.  Thanks!